src/components/InstallTabs/index.tsx

```diff
@@ -79,6 +85,7 @@ const InstallTabs = (): JSX.Element => {
           <Tab key={system.toString()}>{system}</Tab>
         ))}
       </TabList>
+      )}
```

Comment:
It seems to be choking on this https://github.com/nodejs/nodejs.dev/pull/877/checks#step:6:27